PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -112,6 +136,17 @@ log_Check_Env_Var FSLDIR
 HCPPIPEDIR_PostFS="$HCPPIPEDIR/PostFreeSurfer/scripts"
 PipelineScripts="$HCPPIPEDIR_PostFS"
 
+#ExperimentRoot points to actual experiment directory
+case "$LongitudinalMode" in
+    NONE)       ExperimentRoot=$Session; IsLongitudinal=0 ;;
+    TIMEPOINT_STAGE1|TIMEPOINT_STAGE2) 
+                ExperimentRoot="$Session".long."$LongitudinalTemplate"; IsLongitudinal=1 ;;
+    TEMPLATE)   ExperimentRoot="$Subject".long."$LongitudinalTemplate"; IsLongitudinal=1 ;;
+    *)          log_Err_Abort "unrecognized value for --longitudinal mode: $LongitudinalMode" ;;
+esac
```

Comment:
Please use newlines for separate settings and cases, for better readability.  If something isn't obvious at a glance, add a comment, not alignment tricks.
```suggestion
case "$LongitudinalMode" in
    NONE)
        ExperimentRoot=$Session
        IsLongitudinal=0
        ;;
    TIMEPOINT_STAGE1|TIMEPOINT_STAGE2)
        ExperimentRoot="$Session".long."$LongitudinalTemplate"
        IsLongitudinal=1
        ;;
    TEMPLATE)
        #Use subject, not session, for the root
        ExperimentRoot="$Subject".long."$LongitudinalTemplate"
        IsLongitudinal=1
        ;;
    *)
        log_Err_Abort "unrecognized value for --longitudinal mode: $LongitudinalMode"
        ;;
esac
```

Reply:
I suggest to put this on vote.

Reply:
@glasserm @mharms @sivcek @demsarjure Opinions on which of these is more readable/obvious?

Reply:
I much prefer the suggested change.

Reply:
I don't like tons of carriage returns, so I would be fine with the original.

Reply:
It's much easier to quickly digest code if you can just read down for one action at a time, rather than read horizontally and have to determine where a separate action begins on that line.

Reply:
Consistency and obviousness are more important than entertainment value.  Bugs can exist in "trivial" code too, particularly if the formatting is unexpected or just not easy to read.

So far, two reviewers strongly want you to make this change.  glasserm doesn't seem to have a strong opinion.  I don't know that this should wait until the qunex folks choose to respond, and you haven't actually argued that your version has advantages, only that they are similar in your view.  mharms and I disagree with you on them being similar, so please make the change.

Reply:
It depends. But I think this argument became too philosophical for the task to spend more time on it. Done.